ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,82 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def check_mirroring_status(replaying_num=None):
+    """
+    Check mirroring status have expected values
+
+    Args:
+        replaying_num (int): Expected number of images in replaying state
+
+    Returns:
+        bool: True if status contains expected values, False otherwise
+
+    """
+    cbp_obj = ocp.OCP(
+        kind=constants.CEPHBLOCKPOOL,
+        resource_name=constants.DEFAULT_CEPHBLOCKPOOL,
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+    )
+    mirroring_status = cbp_obj.get().get("status").get("mirroringStatus").get("summary")
+    logger.info(f"Mirroring status: {mirroring_status}")
+    keys_to_check = ["health", "daemon_health", "image_health", "states"]
```

Comment:
if "value" doesn't change except for "states" then why is the above keys required ??
if key != states: then expected_value = "OK", otherwise continue/display replaying_num

Reply:
This makes it clear which keys are we going to check in the following lines.